CREATE_YOUR_FIRST_EXTENSION.md

```diff
@@ -2,13 +2,14 @@
 
 * [Initial setup](#initial-setup)
 * [Obtain network address and port information](#obtain-network-address-and-port-information)
-* [Implement the _`Extension`_ interface](#implement-the-extension-interface)
+* [Implement the _`Extension`_ interface](#implement-the-_-extension-_-interface)
```

Comment:
The link to this section is broken.

Reply:
```suggestion
* [Implement the _`Extension`_ interface](#implement-the-extension-interface)
```

Reply:
If you use IDEA, you can generate the TOC automatically :) https://www.jetbrains.com/help/idea/markdown.html#table-of-contents